docs/javascript/v2/guides/javascript-quickstart.mdx

```diff
@@ -43,7 +61,22 @@ const hmsActions = hms.getHMSActions();
 -   `Peer`: A participant in the video call. You are the `local peer` while others are `remote peers`.
```

Comment:
can we come up with a better definition for Room? the current description doesn't give me much